src/SprykerEco/Yves/Heidelpay/Theme/default/views/credit-card-secure/credit-card-secure.twig

```diff
@@ -28,9 +26,7 @@
                             {% include molecule(paymentOption.vars.value, 'Heidelpay') with {
```

Comment:
Why do we need to use different molecules here `paymentOption.vars.value`? Please check.

Reply:
new-registration molecule initialize web-component which loads iframe of heidelpay for new registratration, existing-registration just render existing creditcard info with twig and no ts needed. So it looks like we need different molecules to avoid initing web-component and new-registration logic part in case we don't need it.

Reply:
fixed